addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/Text4ShellScanRuleUnitTest.java

```diff
@@ -119,7 +121,20 @@ void shouldHaveExpectedNumberOfAlertTags() {
         // Given / When
         Map<String, String> alertTags = rule.getAlertTags();
         // Then
-        assertThat(alertTags.size(), is(equalTo(5)));
+        assertThat(alertTags.size(), is(equalTo(8)));
+    }
+
+    @Test
+    void shouldHaveExpectedPolicyAlertTags() {
```

Comment:
We should refactor all the alert tags/policies/mappings verification so they are done consistently throughout the codebase, we have now at least 4 ways (if not more).